lesson_21/homework_21.py

```diff
@@ -0,0 +1,114 @@
+"""Homework 21."""
+
+import logging
+from datetime import datetime
+
+"""
+Моніторингова система клєнта надсилає сигнал,
+що вона працездатна кожні 30-31 сек
+- наприкладTimestamp 05:45:40, а в наступному повідомлені — Timestamp 05:45:09
+(тут різниця heartbeat в 31 секунду)
+
+Є декілька дублючих потоків, що шлють дані одночасно,
+тож ми можемо проаналізувати лише один потік - Key TSTFEED0300|7E3E|0400
+
+Засобами автоматизації проаналізуйте наданий нам лог: hblog.txt
+
+відберіть лише строки з вказаним ключем Key TSTFEED0300|7E3E|0400
+Створіть функцію, що поверне лог-файл, де буде аналіз правильності вимог:
+для кожного випадку де heartbeat більше 31 сек але
+менше 33 логувало WARNING в файл hb_test.log
+для кожного випадку де heartbeat більше рівно 33
+логувало ERROR в файл hb_test.log
+3.Зверніть увагу, що нам для аналізу помилок було
+б добре знати час, в який помилка відбулася.
+
+Обов’язково включіть результат роботи — файл hb_test.log в PR.
+"""
+
+
+class HBAnalyzer:
```

Comment:
Yeap class the best solution here - just better to rename smth aka HBLogAnalyzer